### PR TITLE
Creating initial travis ci build

### PR DESCRIPTION
**Pull request by Reception123**
Copied from orain/ansible-playbook, it is necessary to make some changes

### CODE REVIEW

File: travis.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+language: php
+php:
+  - 5.4
+
+before_install:
+  - sudo FORCE_ADD_APT_REPOSITORY=1 add-apt-repository ppa:rquillo/ansible
```
Comment (JohnFLewis) [inline travis.yml]: not relevant for mw config or at all.

File: travis.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  - sudo apt-get update
+
+install:
+  - sudo apt-get install mw-config
```
Comment (JohnFLewis) [inline travis.yml]: heh?

File: travis.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+before_script:
+  # For the syntax check to work we need some fake vars
+  - mkdir /home/travis/build/Orain/private                -Needs to be changed, Reception123
+  - cp /home/travis/build/Orain/ansible-playbook/.travis.vars.yml /home/travis/build/Orain/private/vars.yml  -^, reception123
```
Comment (JohnFLewis) [inline travis.yml]: above two lines not relevant at all.

File: travis.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+script:
+  # Check playbook syntax
+  - sudo mw-config -i production local.yml --syntax-check
```
Comment (JohnFLewis) [inline travis.yml]: just php syntax checks

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  - sudo apt-get install mw-config
+
+before_script:
+  # For the syntax check to work we need some fake vars
+  - mkdir /home/travis/build/Orain/private                -Needs to be changed, Reception123
+  - cp /home/travis/build/Miraheze/mw-config/.travis.vars.yml /home/travis/build/Miraheze/private/vars.yml  -^, reception123
+
+script:
+  # Check playbook syntax
+  - sudo mw-config -i production local.yml --syntax-check
```
Comment (Southparkfan) [inline .travis.yml]: Remove line 18 and 19

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,36 @@
+language: php
+php:
+  - 5.4
+
+before_install:
```
Comment (Southparkfan) [inline .travis.yml]: Remove line 5-12 (line 5 should be "before_script")

File: .travis.yml [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    channels:
+      - "chat.freenode.net#miraheze"
+    on_success: change
+    on_failure: always
+    skip_join: true
+    template:
+      - "%{repository}/%{branch}/%{commit} - %{author} %{message} %{build_url}"
+  email:
+    recipients:
+      - (Please decide on email)
```
Comment (Southparkfan) [inline .travis.yml]: Should be "- southparkfan@miraheze.org"

**Pull request merged**